contributing.md

```diff
@@ -262,6 +263,107 @@ get aquainted with this development process.
 
 1. **Done, phew.** Once you have met all the requirements, you're code will be merged. Thanks for improving Ignition Robotics!
 
+### Tracking Progress
+
+Ignition development progress is tracked publicly using a GitHub project board.
+Using project boards ensures the community has visibility to what’s coming up,
+external contributors can understand where they can help, and the reasoning behind
+development decisions are visible to everyone.
+
+The [Core development board](https://github.com/orgs/ignitionrobotics/projects/3) can be seen here.
+
+#### Repositories
+
+The following repositories from [Ignition Robotics](https://github.com/ignitionrobotics/) are tracked:
+
+* ign-cmake
+* ign-common
+* ign-fuel-tools
+* ign-gazebo
+* ign-gui
+* ign-launch
+* ign-math
+* ign-msgs
+* ign-physics
+* ign-plugin
+* ign-rendering
+* ign-sensors         
+* ign-tools          
+* ign-transport          
+* docs
+* design
+
+New issues and pull requests, and issue and pull request statuses, from across the tracked repositories are all automatically synced with the same board.
+
+Unfortunately, GitHub boards' cross-org support is poor.
+The following repos can't be handled automatically and have to be manually tracked on the board along with the ones listed above:
+
+* [sdformat]https://github.com/osrf/sdformat
```

Comment:
Did you mean to put the URL into `()` for these to be links?

I'm also wondering if we should add links for the Ignition repos above too